cf-agent/verify_users_pam.c

```diff
@@ -613,10 +615,10 @@ static bool ChangePasswordHashUsingLckpwdf(const char *puser, const char *passwo
     goto unlock_passwd;
 
 close_both:
-    fclose(edit_fd);
+    if (edit_fd != NULL) fclose(edit_fd);
```

Comment:
CONTRIBUTING.md requires us to always use curly braces in if statements

Reply:
yeah, I felt trepidation writing that, should have trusted my gut ;) willfix.